django_mri/serializers.py

```diff
@@ -57,7 +57,24 @@ class ScanSerializer(serializers.HyperlinkedModelSerializer):
 
     class Meta:
         model = Scan
-        fields = "__all__"
+        fields = (
+            "id",
+            "url",
+            "dicom",
+            "subject",
+            "_nifti",
```

Comment:
do you see this will be used in the near future? why is it marked with an underscore?